lib/api/response/auth_response.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:survey_flutter_ic/api/response/converter/response_converter.dart';
+
+part 'auth_response.g.dart';
+
+@JsonSerializable()
+class AuthResponse {
+  final String accessToken;
+  final String tokenType;
+  final int expiresIn;
```

Comment:
For the timestamp, I think it should be `long` instead

Reply:
Oh, yeah, for `timestamp ` It should be `long` type , thanks for reminding @hoangnguyen92dn 🙏

Reply:
@hoangnguyen92dn It seems we don't have type `long` in `dart` 🤔 I think we can use `int` for `timestamp` for more [info](https://stackoverflow.com/a/57139327)  @doannimble May I get you opinion on this 🙏

Reply:
@Wadeewee Oh. `int` has a range for big int in Dart. So we could keep used of `int`.